tests/h/views/api/auth_test.py

```diff
@@ -217,7 +217,7 @@ def oauth_request(self):
     def oauth_provider(self, pyramid_config, auth_client, oauth_request):
         svc = mock.create_autospec(OAuthProviderService, instance=True)
 
-        scopes = [Permission.Annotation.READ, Permission.Annotation.WRITE]
+        scopes = ["annotation:read", "annotation:write"]
```

Comment:
This is a mistake left over for the scope / permission confusion. The reason these tests pass is because the strings are the same in each case.

This is almost certainly why the Enum's didn't work when I tried them, as enum values are not equal to their plain string versions (I think).

Reply:
That's right. Enum values aren't equal to _anything_ outside of the same enum (strings, matching values from other enums, ...) unless you use [`StrEnum`](https://docs.python.org/3.11/library/enum.html#enum.StrEnum) (coming in Python 3.11) or [`IntEnum`](https://docs.python.org/3.4/library/enum.html#enum.IntEnum) (since 3.4)

Reply:
(That's one of the potential benefits of using an enum for permissions, incidentally)